### PR TITLE
quarantinecli 1.0.0 (new formula)

### PR DESCRIPTION
**Pull request by NSAntoine**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/quarantinecli.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+class Quarantinecli < Formula
+  desc "CLI Utility to quarantine and manage quarantined files on macOS"
+  homepage "https://github.com/Serena-io/quarantineCLI"
+  url "https://github.com/Serena-io/quarantineCLI/archive/refs/tags/1.0.0.tar.gz"
+  version "1.0.0"
+  sha256 "69ab41c71ed37c5b47e7dc5ed54bde28f15eaf38c81d804f768f74fb482a28e6"
+  license "MIT"
+
+  depends_on :macos
+  depends_on :xcode => "12.3"
```
Comment (carlocab) [inline Formula/quarantinecli.rb:10]:
```suggestion
  depends_on xcode: "12.3"
```

File: Formula/quarantinecli.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+  def install
+    system "swift", "build", "--disable-sandbox", "-c", "release"
+    bin.install ".build/release/quarantinecli"
+  end
+
+  test do
+    FileUtils.touch("foo")
+    system "#{bin}/quarantinecli", "--quarantine", "foo"
```
Comment (carlocab) [inline Formula/quarantinecli.rb:19]:
```suggestion
    touch "foo"
    system bin/"quarantinecli", "--quarantine", "foo"
```
We don't need interpolation here.

Can we verify that this did something?

File: Formula/quarantinecli.rb [hunk 3]
```diff
@@ -0,0 +1,21 @@
+class Quarantinecli < Formula
+  desc "CLI Utility to quarantine and manage quarantined files on macOS"
```
Comment (carlocab) [inline Formula/quarantinecli.rb:2]:
```suggestion
  desc "Quarantine and manage quarantined files on macOS"
```
We probably don't need to say it's a `CLI Utility`. It's already in the name.

File: Formula/quarantinecli.rb [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+  homepage "https://github.com/Serena-io/quarantineCLI"
+  url "https://github.com/Serena-io/quarantineCLI/archive/refs/tags/1.0.0.tar.gz"
+  version "1.0.0"
+  sha256 "69ab41c71ed37c5b47e7dc5ed54bde28f15eaf38c81d804f768f74fb482a28e6"
+  license "MIT"
+
+  depends_on :macos
```
Comment (carlocab) [inline Formula/quarantinecli.rb:9]: Can we build this on Linux using the `swift` formula?

File: Formula/quarantinecli.rb [hunk 5]
```diff
@@ -0,0 +1,21 @@
+class Quarantinecli < Formula
+  desc "CLI Utility to quarantine and manage quarantined files on macOS"
+  homepage "https://github.com/Serena-io/quarantineCLI"
+  url "https://github.com/Serena-io/quarantineCLI/archive/refs/tags/1.0.0.tar.gz"
+  version "1.0.0"
```
Comment (carlocab) [inline Formula/quarantinecli.rb:5]:
```suggestion
```
This can be read from the `url`, no?

File: Formula/quarantinecli.rb (hunk 2 above)
Comment (Serena-io) [reply Formula/quarantinecli.rb:19]: Is there a way to get the output of a test? If so, yes

File: Formula/quarantinecli.rb (hunk 4 above)
Comment (Serena-io) [reply Formula/quarantinecli.rb:9]: Not at all, the tool uses LSQuarantine from CoreServices, a framework exclusive to Darwin platforms

File: Formula/quarantinecli.rb (hunk 5 above)
Comment (Serena-io) [reply Formula/quarantinecli.rb:5]: Ooh yeah, audit just gave me that, thanks

**Pull request closed without merging**